packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,119 @@
+# MDC Toolbar
+
+MDC Toolbar provides a RTL-aware Material Design toolbar component adhering to the
+[Material Design toolbar spec](https://www.google.com/design/spec/components/toolbars.html)
+Toolbars scroll with content by default, but supports fixed on
+top as well. Currently, this component does not yet support the "Waterfall" or
+"Flexible Header" patterns.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section">
```

Comment:
Should we make this `<section>` to go along with the demo? Also should we change `<div class="mdc-toolbar">` into `<header class="mdc-toolbar">`?